packages/test/test-end-to-end-tests/src/test/pendingOps.spec.ts

```diff
@@ -109,7 +109,9 @@ describeNoCompat("stashed ops", (argsFactory: () => ITestObjectProvider) => {
         assert.strictEqual(await map2.wait(testKey), testValue);
     });
 
-    it("doesn't resend successful op", async function() {
+    // TODO: Remove the skip for the following test once issue #5593 ("doesn't resend successful op" test 
+    // in pendingOps.spec.ts is flaky) has been resolved
```

Comment:
`<soapbox>`I prefer to avoid // TODO comments in the code. Code comments are not a work backlog. Just open the issue and trust that we'll follow through.  If someone's curious why the `skip` is there, they can look at file blame and see this PR.  Or it's a fine compromise to simply say `// Skip this flaky test. See issue #5593`. `</soapbox>`

Reply:
No need to block this PR on this, just taking the opportunity to talk about it

Reply:
Updated the comment with the compromise 😄 keeps it short while also pointing to the issue that is the reason we have the skip